### ISSUE TITLE
Unable to build "master" branch

### ISSUE THREAD

**Issue by SchweinDeBurg** [doxygen/doxygen#7606]
```
[224/290] Building CXX object src/CMakeFiles/_doxygen.dir/sqlite3gen.cpp.o
FAILED: src/CMakeFiles/_doxygen.dir/sqlite3gen.cpp.o
/usr/bin/g++  -DYY_BUF_SIZE=262144 -DYY_READ_BUF_SIZE=262144 -I../qtools -I../libmd5 -I../liblodepng -I../libmscgen -I../libversion -I../vhdlparser -I../src -Igenerated_src -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -march=native -mtune=native -fasynchronous-unwind-tables -mno-aes  -w -I/usr/include -O2 -g -DNDEBUG -fvisibility=hidden -fvisibility-inlines-hidden   -DLLVM_BUILD_GLOBAL_ISEL -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -std=c++11 -std=gnu++14 -MD -MT src/CMakeFiles/_doxygen.dir/sqlite3gen.cpp.o -MF src/CMakeFiles/_doxygen.dir/sqlite3gen.cpp.o.d -o src/CMakeFiles/_doxygen.dir/sqlite3gen.cpp.o -c ../src/sqlite3gen.cpp
../src/sqlite3gen.cpp: In function 'void generateSqlite3ForPage(const PageDef*, bool)':
../src/sqlite3gen.cpp:2489:32: error: 'sectionDict' is not a member of 'Doxygen'
     SectionInfo *si = Doxygen::sectionDict->find(pd->name());
                                ^~~~~~~~~~~
../src/sqlite3gen.cpp:2492:19: error: invalid use of non-static member function 'QCString SectionInfo::title() const'
       title = si->title;
                   ^~~~~
In file included from ../src/outputgen.h:24:0,
                 from ../src/xmlgen.h:18,
                 from ../src/sqlite3gen.cpp:26:
../src/section.h:67:17: note: declared here
     QCString    title()      const { return m_title;      }
                 ^~~~~
ninja: build stopped: subcommand failed.
```

**Comment by albert-github** [comment]
This is a regression on:
```
Commit: 1a56a39b4a97452a5c7c2d8e9d39ab28ca33dff0 [1a56a39]
Commit Date: Friday, February 21, 2020 9:07:13 PM

Restructure section handling
```
when sqlite3 has been enabled for the build.

By disabling the `use_sqlite3` in the CMake phase you can work around the problem (so no sqlite3 support).

I've just pushed a proposed patch, pull request #7607

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.18.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
